.rubocop.yml

```diff
@@ -0,0 +1,3 @@
+---
+AllCops:
+  TargetRubyVersion: 2.3
```

Comment:
I wonder if Rubocop should target the latest version we support, not the earliest...